### PR TITLE
Installation files.

### PR DESCRIPTION
**Pull request by DeadSyntaxdev**
Easier and faster way to install the  needed dependencies for fnf.

### CODE REVIEW

File: Installation.bat [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+haxelib update flixel
+haxelib install flixel
+haxelib install flixel-addons
+haxelib install flixel-ui
+haxelib install hscript
+haxelib install newgrounds
+haxelib git polymod https://github.com/larsiusprime/polymod.git
+haxelib git discord_rpc https://github.com/Aidan63/linc_discord-rpc
+haxelib git flixel-addons https://github.com/HaxeFlixel/flixel-addons
+haxelib git linc_luajit https://github.com/AndreiRudenko/linc_luajit
```
Comment (PrismLight) [inline Installation.bat:17]: Nothing is wrapped inside of a function, this isn't how batch file works.

File: Installation.bat [hunk 2]
```diff
@@ -0,0 +1,17 @@
+haxelib install lime
```
Comment (PrismLight) [inline Installation.bat:1]: Where did the start line go? Where is the function to start this?

**Review by ghost — COMMENTED** [review]
This is not how batchfile works.
This doesn't do anything at all.

File: Installation.bat (hunk 1 above)
Comment (XS09-dev) [reply Installation.bat:17]: > This is not how batchfile works. This doesn't do anything at all.

I literally used it to install the dependencies. It does work like that

File: Installation.bat (hunk 2 above)
Comment (XS09-dev) [reply Installation.bat:1]: > Where did the start line go? Where is the function to start this?

It opens the terminal automatically when you press on it and puts it in. You have to read README.md for the steps on using it and shit.

**Comment by XS09-dev** [comment]
> This is not how batchfile works. This doesn't do anything at all.

Bro it installs the haxe dependencies so you don't have to install them individually. You still have to download haxe 4.2.4 or 5, git, and visual studio.

**Comment by frantastic24** [comment]
> This is not how batchfile works.

it takes two seconds to check and see this is how it works

**Comment by DEMOLITIONDON96** [comment]
> This is not how batchfile works. This doesn't do anything at all.

says the guy who broke my mod's broke almost 5 times, making me or matt fix it

**Comment by XS09-dev** [comment]
> > This is not how batchfile works. This doesn't do anything at all.
> 
> says the guy who broke my mod's code almost 5 times, making me or matt fix it

I would love to know what this dude did.

File: Installation.bat (hunk 1 above)
Comment (Magniill) [reply Installation.bat]: this is working for me actually, now thats what a batchfile works

**Review by notmagniill — APPROVED** [review]
awesome!

File: README.md [hunk 3]
```diff
@@ -2,13 +2,16 @@
...
+1) Download Haxe 4.2.5: https://haxe.org/download/
+2) Download Visual Studio 2019: https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes
+3) Download git-scm: https://git-scm.com
+4) Install all of them.
+5) Install visual Studio 2019 with
+```
+MSVC v142 - VS 2019 C++ x64/x86 build tools
+Windows SDK (10.0.17763.0)
+```
+6) Finally run Installation.bat
```
Comment (ImaginationSuperHero528) [inline README.md:14]: For haxe say latest version.

**Review by rodney528 — APPROVED** [review]

**Comment by XS09-dev** [comment]
> @XS09-dev there's merge conflicts!

Oops I'll fix them now.

**Review by rodney528 — APPROVED** [review]

**Pull request closed without merging**